packages/x-tree-view/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@mui/x-tree-view",
+  "version": "6.0.0-alpha.0",
+  "description": "The community edition of the tree view components (MUI X).",
+  "author": "MUI Team",
+  "main": "src/index.ts",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mui/mui-x/issues"
+  },
+  "homepage": "https://mui.com/x/react-date-tree-view/",
```

Comment:
This looks off to me 🤔

Reply:
Yes, the doc is not live yet, we will have to publish it before actually releasing the package :+1:

Reply:
But will it be 'react-date-tree-view'?

Reply:
Oh you are right :laughing: